### PR TITLE
Make name of portfolio image configurable (#41)

### PR DESCRIPTION
**Pull request by AlexVerrico**
Adds an option to `_config.yaml` to let the user change the filename of the image.
(Also see issue #41 )

### CODE REVIEW

**Comment by AlexVerrico** [comment]
Hi @samarsault 
Any update here?
Cheers,
Alex.

**Comment by samarsault** [comment]
Wouldn't it be better to specify the path relative to the codebase (repo)? I think that leaves less room for confusion.
What do you think?

**Comment by AlexVerrico** [comment]
So the link to the image would be 
``` {{ site.baseurl }}{{site.plainwhite.portfolio_image }}``` ?

**Comment by samarsault** [comment]
> `{{ site.baseurl }}{{site.plainwhite.portfolio_image }}` ?

{{ site.baseurl }} **/** {{site.plainwhite.portfolio_image }}`

**Comment by AlexVerrico** [comment]
Hi @samarsault 
i have made the changes that you suggested, the link to the image is now 
```{{ site.baseurl }}/{{site.plainwhite.portfolio_image }}```
Cheers,
Alex.

File: README.md [hunk 1]
```diff
@@ -54,7 +54,12 @@ plainwhite:
 
 **Updating Placeholder Image**
 
-The placeholder portfolio image can be replaced by the desired image by placing it as `assets/portfolio.png` in your jekyll website.
+The placeholder portfolio image can be replaced by the desired image by placing it as `assets/portfolio.png` in your jekyll website, or by changing the following line in `_config.yaml`
+
+```yaml
+plainwhite:
+  portfolio_image: "portfolio.png" # the filename of the image in /assets/ to display
```
Comment (samarsault) [inline README.md:61]: Could you change this comment? It says assets

Comment (AlexVerrico) [reply README.md]: Hi @samarsault 
I have made the suggested changes.
Cheers,
Alex

**Comment by AlexVerrico** [comment]
@samarsault any update here?

**Comment by samarsault** [comment]
Thanks!

**Pull request merged**